### ISSUE TITLE
Remove Open Y Admin & Provide Upgrade Path to Claro

### ISSUE THREAD

**Issue by ddrozdik** [ymcatwincities/openy#2333]
**Environment:**
Environment: https://sandboxes-d9.openy.org/
Open Y Type: Custom 
Version: 2.7.1

**Description:**
The latest Drupal 9 Open Y version has no tabs in the admin UI. Without tabs, there is no way to find important pages. 
![image](https://user-images.githubusercontent.com/6285570/101830464-972ca600-3b45-11eb-89db-4bd009fe79e2.png)
(the screenshot was made on the Open Y 2.7.0)

Also, breadcrumbs and action links are missing

**Comment by podarok** [comment]
Hi, @ddrozdik 
We have Claro theme in Drupal 9
Tabs are in place
![image](https://user-images.githubusercontent.com/563412/101877672-0f6f8780-3b97-11eb-99f1-a5a1103b8a6a.png)
it was recently fixed by core and tracked in https://github.com/ymcatwincities/openy/issues/2168

I don't think we will have the bandwidth to support all themes for Admin UI - I see you are using the Seven theme. 
@sarah-halby was testing of the Seven theme included in the QA scope?

**Comment by ddrozdik** [comment]
@podarok in our tests we were using the Open Y Admin theme since Claro a bit unstable. But if the Open Y team is going to switch from the Open Y Admin theme to Claro we will need to mark it as deprecated or even delete it from the core.

**Comment by podarok** [comment]
Yes, for sure we'll deprecate Seven, due to it was developed for Drupal 8 ( even the description says that ). 
Here is the communication we had about Claro will be default in Drupal 9. https://openy.slack.com/archives/C6G4ZTLVC/p1599231092155400

Regarding unstable - feel free to share any bugs and concerns for Claro. 
The decision was made due to Claro is way more accessible and mobile-friendly than Seven.

Thank you.

As a comment regarding the Seven theme
I've switched to Seven on a sandbox and found tabs in place
![image](https://user-images.githubusercontent.com/563412/101895507-4c944380-3bb0-11eb-99e6-9faa454e3282.png)
Could you verify on your end?

**Comment by ddrozdik** [comment]
@podarok I confirm tabs are in place for the Seven theme, however in the Open Y Admin, as I mentioned above, tabs are missing. 
![image](https://user-images.githubusercontent.com/6285570/101896840-a568d980-3bba-11eb-8031-ca468b3532ab.png)


But as I see Open Y Admin theme is not used as a default admin theme, so I guess we can ignore this issue and close it.

By the way, do we need an Open Y Admin theme then?

**Comment by ddrozdik** [comment]
@podarok Take a look at this theme https://www.drupal.org/project/gin it was built on the foundation of Claro from one of the lead designers of Claro & Drupal Design System. It is still alpha however looks very promising.

**Comment by sarah-halby** [comment]
Task: remove Open Y Admin Theme & provide upgrade path in the documentation before closing this issue.

**Comment by podarok** [comment]
@ddrozdik thank you for the gin theme suggestion. Our goal is to provide 99% maintenance stability and Claro looks more reliable because supported by Drupal core. Adding a dependency to contrib theme - possible, but we should have more demand, I guess
/cc @sarah-halby

**Comment by sarah-halby** [comment]
Update: to include in February 2021 release or beyond. Notes: we can stay as-is - for associations that use seven for the admin interface, they can keep using it. For new installations, Claro will be the default. If we provide documentation, then the upgrade path will be Claro and there will be a UI change for content managers.
